### PR TITLE
fix(docker): add radiance binaries to PATH

### PR DESCRIPTION
**Pull request by tymokvo**
Add the ol' binaries to the ol' path.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,8 @@ ENV WORKDIR='/home/ladybugbot'
 ENV RUNDIR="${WORKDIR}/run"
 ENV LIBRARYDIR="${WORKDIR}/honeybee-radiance"
 ENV RAYPATH="${WORKDIR}/lib"
```
Comment (mostaphaRoudsari) [inline Dockerfile:8]:
```suggestion
ENV RAYPATH=".:${WORKDIR}/lib"
```

File: Dockerfile [hunk 2]
```diff
@@ -19,8 +20,8 @@ USER ladybugbot
 WORKDIR ${WORKDIR}
 
 # Expects a decompressed radiance folder in the build context
-COPY radiance/usr/local/radiance/bin ${WORKDIR}/bin
-COPY radiance/usr/local/radiance/lib ${WORKDIR}/lib
+COPY radiance/usr/local/radiance/bin ${BINPATH}
+COPY radiance/usr/local/radiance/lib ${RAYPATH}
```
Comment (mostaphaRoudsari) [inline Dockerfile:24]: Sorry for making it not as good looking by adding the current directory to RAYPATH.

```suggestion
COPY radiance/usr/local/radiance/lib ${WORKDIR}/lib
```

**Comment by tymokvo** [comment]
:arrow_up: These make much more sense together. Thank you @mostaphaRoudsari !

File: Dockerfile [hunk 3]
```diff
@@ -19,8 +20,8 @@ USER ladybugbot
...
 
 # Install honeybee-radiance
 COPY honeybee_radiance ${LIBRARYDIR}/honeybee_radiance
```
Comment (mostaphaRoudsari) [inline Dockerfile:27]: @tymokvo, one more comment here! Does this compile the entrypoints for running honeybee-radiance commands from the command line? Don't we need to use pip install to create the executable files under Scripts folder?

I'm lazy to build the docker and test it myself but what happens when you run `honeybee-radiance --help` from the command line? If it gives you the right output then ignore my comment here.

**Review by mostaphaRoudsari — COMMENTED** [review]
Sorry! Added one more comment.

File: Dockerfile (hunk 3 above)
Comment (tymokvo) [reply Dockerfile:27]: Yep, the cli help prints out. I believe `pip` handles that installation on line 38.

File: Dockerfile (hunk 2 above)
Comment (tymokvo) [reply Dockerfile]: It's ok :crying_cat_face:

**Comment by tymokvo** [comment]
I thought from your description that `RAYPATH` needed to be removed from `PATH` so I added a new commit instead of accepting the suggestions. Even though they seem perfectly good to me!

File: Dockerfile (hunk 3 above)
Comment (mostaphaRoudsari) [reply Dockerfile]: You are right. Didn't see the existing lines on my phone. :|

**Review by mostaphaRoudsari — APPROVED** [review]
Looks great!

**Pull request merged**